README.md

```diff
@@ -1,2 +1,44 @@
-# Mobile-first-D3
-Mobile First approach
+# PORTFOLIO - MICROVERSE TASK
+
+> This a "mobile-first" approach, that gives more priorities in developing mobile version first and perfecting the mobile user experience. Due to the fact that there are rising numbers of mobile users as compared to the desktop users.
+
+## Built With
+
+- Html,Css,
+- Linters
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+All you need VScompiler or a compiler of your choice.
+
+### Setup
+
+You will need to clone use git clone command (git clone url repo) and open the html with live server to view.
```

Comment:
- [ ] **[OPTIONAL]** It could be better to list the process, some things could be obvious but anybody could read your README file. I recommend you this 👇🏽 

- Clone the repo: `git clone git@github.com:felixDev22/Mobile-first-D3.git`
- Change to the feature branch: `git checkout hero-section`
- Open the `index.html` in your browser.

Reply:
Thank you for the clear and informative feedback